plugins/jobtype/jobtypes/hadoopShell/private.properties

```diff
@@ -0,0 +1,3 @@
+jobtype.class=azkaban.jobtype.HadoopShell
+
+command.blacklist.regex=(?i).*kinit.*
```

Comment:
actually this will become a bug if user do something like this - 

hadoop fs  XXX ....       # Note: no need to do kinit as the powerful azkaban will take care of this.  


The whole like will be filtered as the comment contains blacklist words.